scripts/collector/postgres/sql/database_details.sql

```diff
@@ -0,0 +1,109 @@
+with db as (
+    select db.oid as database_oid,
+        db.datname as database_name,
+        db.datconnlimit as max_connection_limit,
+        db.datistemplate as is_template_database,
+        pg_encoding_to_char(db.encoding) as character_set_encoding,
+        pg_database_size(db.datname) as total_disk_size_mb
+    from pg_database db
+),
+db_size as (
+    select s.datid as database_oid,
+        s.datname as database_name,
+        s.numbackends as backends_connected,
+        s.xact_commit as txn_commit_count,
+        s.xact_rollback as txn_rollback_count,
+        s.blks_read as blocks_read_count,
+        s.blks_hit as blocks_hit_count,
+        s.tup_returned as tup_returned_count,
+        s.tup_fetched as tup_fetched_count,
+        s.tup_inserted as tup_inserted_count,
+        s.tup_updated as tup_updated_count,
+        s.tup_deleted as tup_deleted_count,
+        s.conflicts as query_conflict_count,
+        s.temp_files as temporary_file_count,
+        s.temp_bytes as temporary_file_bytes_written,
+        s.deadlocks as detected_deadlocks_count,
+        s.checksum_failures as checksum_failure_count,
+        s.checksum_last_failure as last_checksum_failure,
+        s.blk_read_time as block_read_time_ms,
+        s.blk_write_time as block_write_time_ms,
+        s.session_time as session_time_ms,
```

Comment:
line 31-37 do not exist in the default debian postgresql i've installed
so this returns nothing instead of database and size

Reply:
I think that only captures data when a certain extension is enabled.  Let me grab the details that I was reading yesterday and send them over to you.